src/facilities-context/transforms.tsx

```diff
@@ -106,7 +106,7 @@ function mergeModelVersions({
   userVersions,
   referenceFacility,
 }: Optional<MergedHistoryInputs, "referenceFacility">): ModelInputs[] {
-  const combinedVersions: ModelInputs[] = [...userVersions];
+  const combinedVersions: ModelInputs[] = [...userVersions.filter(hasCases)];
```

Comment:
I was also considering saving the reference facility's transformed covidCases data directly to the newly mapped userFacility (from `SyncNoUserFacilities` line 258. That way we wouldn't have to filter out the "fake" modelInputs and the userFacility created from the referenceFacility would already have the correct modelInputs and versions. Do you think that would make sense?

Reply:
hm yeah, I thought about doing that ... I am not dead-set against it necessarily but there were a couple of issues I saw that made this seem like an easier fix, at least within the scope of the bug I was looking at: 

1. by default it sets the current day as `observedAt`, which is likely to be wrong. it should be set to whatever the most recent observed at date is in reference data. not sure how much work that would be to fix
2. i was a little leery of putting data into the user's facilities collection that they didn't actually enter themselves. In practice it's probably not a huge deal but it did seem slightly unclean somehow. We could do it if it's really necessary for some reason, but as far as I could tell for this issue anyway, it's not?

what do you think? is there something I'm missing or something relevant to your ticket that you think might indicate a different fix here?

Reply:
I need to do some more testing but my initial hunch was that the rtData calculation fetches were firing first with the incorrect modelInputs, and they were being saved as errors after the second round of rtData calculations were requested and returned. I think this fix makes sense for this issue, I can keep digging on the rtData calculations for the referenceFacilities from this modal tomorrow morning.

Reply:
sounds plausible ... FWIW i didn't really see difference in that behavior with this change